cinema/views.py

```diff
@@ -34,6 +39,10 @@ class MovieViewSet(viewsets.ModelViewSet):
     queryset = Movie.objects.all()
     serializer_class = MovieSerializer
 
+    @staticmethod
+    def _params_to_int(params):
+        return [int(str_id) for str_id in params.split(",")]
```

Comment:
You could place this method in a mixin and reuse it in this class and in the `MovieSessionViewSet` as well to avoid further code duplication